e2e/test/scenarios/question/reproductions/22230-query-builder-aggregation-filtering.cy.spec.js

```diff
@@ -0,0 +1,43 @@
+import {
+  restore,
+  popover,
+  summarize,
+  openPeopleTable,
+  visualize,
+} from "e2e/support/helpers";
+
+describe("issue 22230", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should allow filtering on aggregated fields", () => {
+    openPeopleTable({ mode: "notebook" });
+
+    summarize({ mode: "notebook" });
+    popover().findByRole("option", { name: "Maximum of ..." }).click();
+    // I don't know why this can't be chained, I tried it and it didn't work.
+    popover().findByRole("option", { name: "Name" }).click();
```

Comment:
I've tried chaining this call like this:
```ts
popover()
  .findByRole('option', { name: 'Maximum of ...' })
  .click()
  .findByRole('option', { name: 'Name' })
  .click()
```

But it doesn't work. Not sure why.

Reply:
I think the second `findByRole` searches for an element within the first option, not within the popover

Reply:
Ahh I see. That makes sense now!